api.ts

```diff
@@ -958,14 +959,26 @@ export function isSyncPackFormula(fn: BaseFormula<ParamDefs, any>): fn is Generi
  * are called repeatedly until there is no continuation returned.
  */
 export interface SyncFormulaResult<K extends string, L extends string, SchemaT extends ObjectSchemaDefinition<K, L>> {
-  /** The list of results from this page. */
+  /** The list of rows from this page. */
   result: Array<ObjectSchemaDefinitionType<K, L, SchemaT>>;
+
   /**
    * A marker indicating where the next sync formula invocation should pick up to get the next page of results.
    * The contents of this object are entirely of your choosing. Sync formulas are called repeatedly
    * until there is no continuation returned.
    */
   continuation?: Continuation;
+
+  /**
+   * Once there is no additional continuation returned from a pack sync formula, this may be returned instead, to give
+   * metadata about the entirety of the sync execution.
+   *
+   * This is ignored if there is also a {@link continuation} on this object.
+   *
+   * TODO(patrick): Unhide this
+   * @hidden
+   */
+  syncExecutionCompletionMetadata?: SyncExecutionCompletionMetadata;
```

Comment:
Nitpick: The key name seems overly verbose. Would `metadata` suffice? Or perhaps `completion`?

Reply:
I like how `completion` matches `continuation`, I'll use that.